src/frontend/js/components/Modal/index.spec.tsx

```diff
@@ -50,4 +55,50 @@ describe('<Modal />', () => {
     expect($modal).toBeInstanceOf(HTMLDivElement);
     expect($overlay).toBeInstanceOf(HTMLDivElement);
   });
+
+  it('displays close button', () => {
+    render(
+      <IntlProvider locale="en">
+        <Modal isOpen={true} />
+      </IntlProvider>,
+    );
+
+    const $closeButton = document.querySelector('.modal .modal__closeButton');
+    expect($closeButton).toBeInstanceOf(HTMLElement);
```

Comment:
```suggestion
We are using [methods provided](https://testing-library.com/docs/queries/about#types-of-queries) by `testing-library` to select elements with accessibility in mind.

```suggestion
    // A close button should be displayed
    screen.getByRole('button', { name: 'Close dialog' });
```
```